### PR TITLE
Add titleBarStyle: hidden to Electron Window and some Trusted UI initial styles

### PR DESCRIPTION
**Pull request by diogoperillo**
https://www.dropbox.com/s/f9j75q3z2qmajs1/Screenshot%202019-01-24%2009.39.48.png?dl=0

### CODE REVIEW

File: packages/launcher/src/renderer/app/AppContainer.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import type { ID } from '@mainframe/utils-id'
 import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native-web'
 
+import styled from 'styled-components/native/'
```
Comment (PaulLeCam) [inline packages/launcher/src/renderer/app/AppContainer.js]: Looks like there is an extra trailing `/`

File: packages/launcher/src/main/index.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ const newWindow = (params: Object = {}) => {
     width: params.width || 800,
     height: params.height || 600,
     show: false,
+    titleBarStyle: 'hidden',
```
Comment (PaulLeCam) [inline packages/launcher/src/main/index.js]: Has this been tested on Window and Linux?

Comment (diogoperillo) [reply packages/launcher/src/main/index.js]: not really, but as I read, this will only make changes to mac no?

Comment (diogoperillo) [reply packages/launcher/src/main/index.js]: I don't have a way to test in linux or windows? do you know how I can do it?

Comment (PaulLeCam) [reply packages/launcher/src/main/index.js]: If it only affects macOS that's fine, but better make sure the UX is consistent on Linux and Windows. You can use VMs for example with VirtualBox to check on these systems.

Comment (diogoperillo) [reply packages/launcher/src/main/index.js]: K I'll take a look

File: packages/launcher/src/renderer/app/AppContainer.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import type { ID } from '@mainframe/utils-id'
 import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native-web'
 
+import styled from 'styled-components/native'
+
 import colors from '../colors'
 import Text from '../UIComponents/Text'
```
Comment (avclarke) [inline packages/launcher/src/renderer/app/AppContainer.js]: We should probably update these to use Morpheus UI and maybe change the views with styles also to styled components so we don't have a mix of the two types.

File: packages/launcher/src/renderer/app/AppContainer.js [hunk 4]
```diff
@@ -7,6 +7,8 @@ import type { ID } from '@mainframe/utils-id'
 import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native-web'
 
+import styled from 'styled-components/native/'
+
 import colors from '../colors'
 import Text from '../UIComponents/Text'
```
Comment (avclarke) [inline packages/launcher/src/renderer/app/AppContainer.js]: We should probably update these to use Morpheus UI and maybe change the views with styles also to styled components so we don't have a mix of the two types.

Comment (diogoperillo) [reply packages/launcher/src/renderer/app/AppContainer.js]: Yeah I was going to do it later

File: packages/launcher/src/main/index.js (hunk 2 above)
Comment (diogoperillo) [reply packages/launcher/src/main/index.js]: Works well in Windows too @PaulLeCam no issues.

**Pull request merged**